spec/freebsd/interface.go

```diff
@@ -2,14 +2,16 @@
 
 package freebsd
 
-import "github.com/mackerelio/mackerel-agent/spec"
+import (
+	mkr "github.com/mackerelio/mackerel-client-go"
```

Comment:
We would like to use mackerel alias, because mackerel-client-go is not mkr. It's ok until it conflicts the mackerel package inside this repository.